scripts/suite_objects.py

```diff
@@ -1143,6 +1144,9 @@ def analyze(self, phase, group, scheme_library, suite_vars, level):
             if phase in func:
                 my_header = func[phase]
```

Comment:
This just says `phase`, is `phase` always `run`?

Reply:
This is to catch instances when "func" doesn't contain a run.
But yeah, phase can be init, run, or final, but we don't care if init and final are not present.

Reply:
I still don't understand the logic. Somewhere, the code  calls `analyze` for a certain phase (which can be anything you said: `init`, `final`, ...). Then in line 1144, you check if that phase is in the func dictionary. If it is not, you set `has_run_phase` to `False`. That makes sense if `phase` is `run`, but not if the argument `phase` is `init`?

Reply:
See #259 for more on this topic.

Reply:
Yes, so that reminds me that physics called from the FV3 dycore need their own initialization phase and that can't be called at the same time as the rest of the physics. I went back to the old thread and read that one of the disadvantages of being able to do this in prebuild is that suites are host specific; well, we have determined by now that this is always going to be the case. There is not a single suite in the existing ccpp-framework and physics that works across just two models.

So, in short, we'll need to figure this out or we won't be able to do the fast physics init required for GFDL-MP in the UFS.

Reply:
There's another option though. We could try to treat the FV3 dycore with the fast physics as its own "host model" for CCPP framework purposes. I thought about this in the past, when I heard discussions about moving more physics into the FV3 dycore ("inline physics"). I think that should be possible with relatively little engineering on the build system side.

Reply:
If you could, yes please. I don't remember. Thanks very much!

Reply:
I have added what I recall to the description section of #275 .

Reply:
Thanks! Yes, I think this would give us the desired capability.